interfaces/Portalicious/src/app/services/rtl-helper.service.ts

```diff
@@ -20,12 +20,27 @@ const LOGICAL_VALUE_TO_PHYSICAL_VALUE_MAP: Record<
   },
 };
 
+const LTR_VALUE_TO_RTL_VALUE_MAP: Record<
+  Direction,
+  Record<PhysicalPosition, PhysicalPosition>
+> = {
+  ltr: {
+    left: 'left',
+    right: 'right',
+  },
+  rtl: {
+    left: 'right',
+    right: 'left',
+  },
```

Comment:
```suggestion
  ltr: {
    backward: 'left',
    forward: 'right',
  },
  rtl: {
    backward: 'right',
    forward: 'left',
  },
```

To fully get rid of literal using "left" or "right" from our thinking, I'd like to use something like this... (But `start`/`end` would also be fine with me... but there's more room for confusion, I'd think?)